deploy/values.yaml

```diff
@@ -0,0 +1,44 @@
+CHECK_KYVERNO_POLICIES: NO
+webhooktlsGetter:
+  image:
+    registry: gcr.io
+    repository: cloudark-kubeplus/webhook-tls-getter
+    tag: latest
```

Comment:
Since this is the top-level values.yaml, we should have the tags that are publicly available. We can add a section in contributing guide mentioning that change this file to use the 'latest' tags during development.